app/src/main/java/com/mtesitoo/helper/ImageHelper.java

```diff
@@ -0,0 +1,45 @@
+package com.mtesitoo.helper;
+
+import android.content.Context;
+import android.graphics.Bitmap;
+import android.graphics.Canvas;
+import android.graphics.Color;
+import android.graphics.Paint;
+import android.support.v4.graphics.drawable.RoundedBitmapDrawable;
+import android.support.v4.graphics.drawable.RoundedBitmapDrawableFactory;
+
+/**
+ * Created by eduardodiaz on 04/09/2017.
+ */
+
+public class ImageHelper {
```

Comment:
looks fine to me code wise, however haven't run UI on my device. I'm assuming its similar to mockups.

Reply:
Yes, it's just like the mockup but we can wait till you run it